pandas/core/series.py

```diff
@@ -858,9 +858,14 @@ def ravel(self, order: str = "C") -> ArrayLike:
         Examples
         --------
         >>> s = pd.Series([1, 2, 3])
-        >>> s.ravel()
+        >>> s.ravel()  # doctest: +SKIP
         array([1, 2, 3])
         """
+        warnings.warn(
+            "Series.ravel is deprecated. Use numpy.ravel directly.",
```

Comment:
Or maybe we should also point to `to_numpy()` in case people use this to essentially get the numpy series (the "ravel" aspect never does something, since Series is already 1D)

Reply:
Yes and yes

Reply:
> or just "dont do it bc it isnt necessary"?

That's not necessarily correct. If you rely on the current return value, i.e. if your code requires that the object is a 1D numpy array, and you currently use `ravel` for that, you _do_ need to replace ravel with a conversion to numpy.  
(and that's actually how we were implicitly using it internally)